src/features/hotspots/setup/HotspotSetupExternalConfirmScreen.tsx

```diff
@@ -189,7 +189,7 @@ const HotspotSetupExternalConfirmScreen = () => {
         mode="contained"
         variant="primary"
         onPress={navNext}
-        disabled={publicKey !== address}
+        disabled={ownerAddress !== address}
```

Comment:
So this was already an existing bug right? Just should have been owner instead of hotspot address?